dags/intacct_historical.py

```diff
@@ -75,7 +76,8 @@
             connector_id=connector_id,
             poke_interval=30,
             execution_timeout=timedelta(hours=6),
-            retries=0,
```

Comment:
I know I'm a bit late to the party, but would it still make sense to have a fixed number of retries to avoid the task silently failing? Unless I misunderstood the change.

Reply:
All tasks in this DAG have `retries=2` by default, as configured in the `default_args`.

Reply:
Thanks for confirming! 🙇